thavalon-server/src/notifications/mod.rs

```diff
@@ -0,0 +1,59 @@
+//! Module containing functions to send notifications to a user. This can
+//! include email, text, or other forms of communication. This module should be
+//! used for sending non-game related notifications to a player, such as account
+//! notifications.
+
+pub mod account;
+mod errors;
+
+pub use errors::NotificationError;
+use lazy_static::lazy_static;
+use mailgun_rs::{EmailAddress, Mailgun, Message};
+use std::env;
+
+const SMTP_DOMAIN: &str = "mg.bennavetta.com";
+const SMTP_USER: &str = "thavalon@mg.bennavetta.com";
```

Comment:
With the HTTP API, I *think* sending from `no-reploy@mg.bennavetta.com` will also work